client/src/components/MediaCard/MediaCard.stories.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import MediaCard from './MediaCard';
+import books from '../../../../assets/images/media-card-1.png'
+import happiness from '../../../../assets/images/media-card-2.png';
+import van from '../../../../assets/images/media-card-3.png';
+import nightSky from '../../../../assets/images/media-card-4.png';
+import waterfall from '../../../../assets/images/media-card-5.png';
+import flight from '../../../../assets/images/media-card-6.png';
+
+storiesOf('MediaCard', module)
+  .add("Books", () => <MediaCard imageSrc={books} userFirstName='User' />)
+  .add("Happiness", () => <MediaCard imageSrc={happiness} userFirstName='User' />)
+  .add("Van", () => <MediaCard imageSrc={van} userFirstName='User' />)
+  .add("Night Sky", () => <MediaCard imageSrc={nightSky} userFirstName='User' />)
+  .add("Waterfall", () => <MediaCard imageSrc={waterfall} userFirstName='User' />)
+  .add("Flight", () => <MediaCard imageSrc={flight} userFirstName='User' />)
```

Comment:
Instead of adding a property for the `userFirstName`, try including a property for `footerText`. Then, you can pass the user's name in as a prop. We don't know what the user object will look like yet, so it makes sense that you put a placeholder there, but we can add a placeholder that is more dynamic than a string. For example, `footerText={`Posted by: ${this.state.user}`}`.